var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -49,117 +52,239 @@ class Gdal(AutotoolsPackage):
-    variant('liblzma',   default=True,  description='Include liblzma support')
-    variant('pg',        default=False, description='Include PostgreSQL support')
-    variant('cfitsio',   default=False, description='Include FITS support')
-    variant('png',       default=False, description='Include PNG support')
-    variant('jpeg',      default=True,  description='Include JPEG support')
-    variant('gif',       default=False, description='Include GIF support')
-    variant('sosi',      default=False, description='Include SOSI support')
-    variant('hdf4',      default=False, description='Include HDF4 support')
-    variant('hdf5',      default=False, description='Include HDF5 support')
-    variant('kea',       default=False, description='Include kealib')
-    variant('netcdf',    default=False, description='Include netCDF support')
-    variant('jasper',    default=False, description='Include JPEG-2000 support via JasPer library', when='@:3.4')
-    variant('openjpeg',  default=False, description='Include JPEG-2000 support via OpenJPEG 2.x library')
-    variant('xerces',    default=False, description='Use Xerces-C++ parser')
-    variant('expat',     default=False, description='Use Expat XML parser')
-    variant('libkml',    default=False, description='Use Google libkml')
-    variant('odbc',      default=False, description='Include ODBC support')
-    variant('curl',      default=False, description='Include curl')
-    variant('xml2',      default=False, description='Include libxml2')
-    variant('sqlite3',   default=False, description='Use SQLite 3 library')
-    variant('pcre2',     default=False, description='Include libpcre2 support', when='@3.4.1:')
-    variant('pcre',      default=False, description='Include libpcre support')
-    variant('geos',      default=False, description='Include GEOS support')
-    variant('qhull',     default=False, description='Include QHull support')
-    variant('opencl',    default=False, description='Include OpenCL (GPU) support')
-    variant('poppler',   default=False, description='Include poppler (for PDF) support')
-    variant('proj',      default=True,  description='Compile with PROJ.x')
-    variant('perl',      default=False, description='Enable perl bindings', when='@:3.4')
-    variant('python',    default=False, description='Enable python bindings')
-    variant('java',      default=False, description='Include Java support')
-    variant('mdb',       default=False, description='Include MDB driver', when='@:3.4 +java')
-    variant('armadillo', default=False, description='Include Armadillo support for faster TPS transform computation')
-    variant('cryptopp',  default=False, description='Include cryptopp support')
-    variant('crypto',    default=False, description='Include crypto (from openssl) support')
-    variant('grib',      default=False, description='Include GRIB support')
+    version('2.4.4', sha256='a383bd3cf555d6e1169666b01b5b3025b2722ed39e834f1b65090f604405dcd8')
```

Comment:
(question) It looks like `2.4.4` and prior versions when from being `deprecated` to not: was that intended?

Reply:
Yes, that was intentional. I originally planned on removing GDAL 2 in order to use driver variants in #30916, but this alternative PR supports GDAL 2 and 3.